l10n_it_fiscal_document_type/models/fiscal_document_type.py

```diff
@@ -4,20 +4,20 @@
 class FiscalDocumentType(models.Model):
     _name = 'fiscal.document.type'
 
-    code = fields.Char(string='Codice', size=5)
-    name = fields.Char(string='Nome', size=100)
-    out_invoice = fields.Boolean(string='Fattura vendita')
-    in_invoice = fields.Boolean(string='Fattura acquisto')
-    out_refund = fields.Boolean(string='Nota di credito')
-    in_refund = fields.Boolean(string='Nota di debito')
+    code = fields.Char(string='Code', size=5)
+    name = fields.Char(string='Name', size=100)
+    out_invoice = fields.Boolean(string='Sales Invoice')
+    in_invoice = fields.Boolean(string='Purchase Invoice')
+    out_refund = fields.Boolean(string='Credit Note')
+    in_refund = fields.Boolean(string='Debit Note')
```

Comment:
I termini odoo base sarebbero
```
            ('out_invoice','Customer Invoice'),
            ('in_invoice','Vendor Bill'),
            ('out_refund','Customer Credit Note'),
            ('in_refund','Vendor Credit Note'),
```

Reply:
Io mi ero limitato a tradurre i termini che avevo trovato in italiano. :-)
Adesso correggo.

Reply:
Infatti c'era un problema di definizioni alla base del modulo, risolto poi con https://github.com/OCA/l10n-italy/pull/595